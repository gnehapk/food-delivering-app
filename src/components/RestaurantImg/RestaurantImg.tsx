import React from 'react';

const RestaurantImg: React.FC<RestaurantImgPropsType> = (props) => (
  <img src={props.restaurantImg} alt="RestaurantImg" />
);

type RestaurantImgPropsType = {
  restaurantImg: string;
};

export default RestaurantImg;