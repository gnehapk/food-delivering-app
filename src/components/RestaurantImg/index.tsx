import RestaurantImg from './RestaurantImg';

export default RestaurantImg;