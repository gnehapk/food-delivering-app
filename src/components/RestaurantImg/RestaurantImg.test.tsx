import React from 'react';
import TestRenderer from 'react-test-renderer';
import RestaurantImg from './RestaurantImg';

describe('RestaurantImg', () => {
  it('should render correctly', () => {
    const tree = TestRenderer.create(<RestaurantImg restaurantImg="kithcen.png"/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

