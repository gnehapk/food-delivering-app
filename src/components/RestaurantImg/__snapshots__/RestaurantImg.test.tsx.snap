// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestaurantImg should render correctly 1`] = `
<img
  alt="RestaurantImg"
  src="kithcen.png"
/>
`;
